App.Stocks/Repositories/App_StocksRepository.cs

```diff
@@ -0,0 +1,71 @@
+﻿using App.Configuration;
+using App.Repositories;
+using App.Stocks.ModelsView;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace App.Stocks.Repositories
+{
+	public class App_StocksRepository : IStocksRepository, ITransientDependency
```

Comment:
1) Invalid naming 
2) invalid class content (not correspond to class name)
3) class is not in use ?